src/pathresolver.ts

```diff
@@ -84,7 +87,7 @@ class PathResolver {
     }
     const absUrl = url.resolve(importUrl, relUrl);
     if (this.abspath) {
-      return url.resolve('/', absUrl);
+      return url.resolve(this.abspath, absUrl);
```

Comment:
This should all happen after all the inlining, or it will get confusing to resolve in cases where the abspath contains a protocol and domain.